CONTRIBUTING.rst

```diff
@@ -81,36 +75,26 @@ Building the Documentation
 --------------------------
 
 If you make changes to the documentation,
-you can re-create the HTML pages using Sphinx_.
-You can install it and a few other necessary packages with::
+you can re-create the HTML pages using Sphinx_::
 
-   pip install -r docs/requirements.txt
-
-To create the HTML pages, use::
-
-   python -m sphinx docs/ build/sphinx/html -b html
+   uv run python -m sphinx docs/ build/html -b html
```

Comment:
I tried it on a freshly installed system and got the following warning:

```
WARNING: dot command 'dot' cannot be run (needed for graphviz output), check the graphviz_dot setting
```

resulting in:

![image](https://github.com/user-attachments/assets/df81c0e1-3324-45d8-b6d5-62ed4bba4805)

So maybe we should mention that `graphviz` needs to be installed to properly render dot files?

Reply:
I added a comment to CONTRIBUTING that `graphivz` is needed.